DeploymentCloud/Deployment.Common/Resources/Templates/Resource-Template.json

```diff
@@ -812,6 +833,88 @@
 			"dependsOn": [
 				"[resourceId('Microsoft.Web/sites', parameters('sites_web_name'))]"
 			]
+		},
+		{
+            "apiVersion": "2015-08-31-preview",
+            "name": "[parameters('sparkManagedIdentity')]",
+			"location": "[parameters('default_resource_location')]",
+            "type": "Microsoft.ManagedIdentity/userAssignedIdentities",
+            "properties": {}
+        },
+		{
+			"type": "Microsoft.Storage/storageAccounts",
```

Comment:
are we creating a blob here?  Is it having all the right settings regarding anonymous access etc

Reply:
yes. we moved the storage account creation step from HDInsight creation step to common resources creation step. The settings are all the same.